support-frontend/assets/helpers/productCatalog.tsx

```diff
@@ -312,3 +312,33 @@ export const productCatalogDescription: Record<ProductKey, ProductDescription> =
 			},
 		},
 	};
+
+export const productCatalogDescriptionNewBenefits: Record<
+	ProductKey,
+	ProductDescription
+> = {
+	...productCatalogDescription,
+	TierThree: {
+		...productCatalogDescription.TierThree,
+		benefits: [
+			...productCatalogDescription.TierThree.benefits,
+			{
+				copy: 'Newspaper archive',
+				isNew: true,
+			},
+		],
+	},
+	SupporterPlus: {
+		...productCatalogDescription.SupporterPlus,
+		benefits: [
+			...productCatalogDescription.SupporterPlus.benefits,
+			{
+				copy: 'Unlimited access to the Guardian Feast App',
+				isNew: true,
+				tooltip:
+					'Make a feast out of anything with the Guardian’s new recipe app. Feast has thousands of recipes including quick and budget-friendly weeknight dinners, and showstopping weekend dishes – plus smart app features to make mealtimes inspiring.',
+			},
+		],
+		offers: [],
+	},
+};
```

Comment:
I know this is a pattern we've had before, but I've found it a little cumbersome. Maybe we could just add it to the catalog with a `shouldRender` method that takes the switches and whatever data to calculate that?